modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpFileElement.cpp.mustache

```diff
@@ -82,7 +82,7 @@ bool {{prefix}}HttpFileElement::fromJsonValue(const QJsonValue &jval) {
         file.remove();
     }
     result = file.open(QIODevice::WriteOnly);
-    file.write(QJsonDocument(jval.toObject()).toBinaryData());
+    file.write(QJsonDocument(jval.toObject()).toJson());
```

Comment:
Is this backward compatible?
`toJson` is deprecated only in 5.15

Reply:
Tested in 5.9 LTS

Reply:
Added another version check since build failed.